scripts/data_guide/generate_data_guide.py

```diff
@@ -0,0 +1,87 @@
+"""
+This script generates a data guide for a given release and project ID.
+
+Usage: python generate_data_guide.py <release> <project_id>
+"""
+import sys
+import os
+import subprocess
+
+from jinja2 import Template
+import pandas as pd
+from synapseclient import Synapse, File
+
+# Login to Synapse
+syn = Synapse()
+syn.login()
+
+# Parse command-line arguments
+release = sys.argv[1]
+project_id = sys.argv[2]
+
+# Function to get the release folder Synapse ID
+def get_release_folder_synid(database_synid_mappingid: str, release: str) -> str:
+    """
+    Given a Synapse ID mapping database and a release name, returns the Synapse ID of the release folder.
+
+    Args:
+        database_synid_mappingid: The Synapse ID of the database mapping Synapse IDs to database names
+        release: The name of the release to retrieve the Synapse ID for
+
+    Returns:
+        The Synapse ID of the release folder
+
+    Raises:
+        ValueError: If the given release name is not found in the database
+    """
+
+    mapping_df = syn.tableQuery(f"SELECT * FROM {database_synid_mappingid}").asDataFrame()
```

Comment:
Does it matter if the results of the tableQuery get left on the disk where the script was run? I had been doing this logic to make sure the results get deleted:

https://github.com/Sage-Bionetworks-Workflows/orca-recipes/blob/be11f73dd718f690309be0f4763405dc511e5dda/dags/synapse-dataset-to-croissant.py#L683-L689